sdk/resourcemanager/Azure.ResourceManager/samples/Sample4_MultiTenant.md

```diff
@@ -0,0 +1,96 @@
+Example: Authenticate across tenants
```

Comment:
This example is not only about authenticate across tenants (the virtual network peering part), could we change the title to emphasize that? Or separate that to another example.
I believe, to enable the `cross-tenant authentication`, we only need the policy, put it in options and use it to create a `ArmClient` and we have done the "cross tenant authentication".
The rest part is an example for "how cross tenant resources work", not the authentication itself

Reply:
Removed irrelevant part